app/gui2/src/components/documentation/DocsSynopsis.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import iconImportant from '@/assets/icon-important.svg'
+import iconInfo from '@/assets/icon-info.svg'
+import type { Doc } from '@/util/docParser'
+
+const props = defineProps<{ sections: Doc.Section[] }>()
+</script>
+
+<template>
+  <!-- eslint-disable vue/no-v-html -->
+  <div v-if="props.sections.length > 0" class="sectionContent">
+    <div v-for="(section, i) in props.sections" :key="i">
+      <p v-if="'Paragraph' in section" class="paragraph">
+        <span v-html="section.Paragraph.body ?? 'Invalid body'"></span>
```

Comment:
The `??` operator in templates makes syntax highlighting break. Here it would be fine to use `||` instead.

Reply:
Fixed